### PR TITLE
feat: CCIE-1661: Distribute HVM via Brew

### PR DESCRIPTION
**Pull request by dspadea**
<details open>
  <summary><a href="https://czi-tech.atlassian.net/browse/CCIE-1661" title="CCIE-1661" target="_blank">CCIE-1661</a></summary>
  <br />
  <table>
    <tr>
      <th>Summary</th>
  <td>Distribute HVM via Homebrew</td>
    </tr>
    <tr>
      <th>Type</th>
      <td>
        Task
        Task
      </td>
    </tr>
    <tr>
      <th>Status</th>
      <td>In Review</td>
    </tr>
  </table>
</details>

---

Make it possible to install hvm via Homebrew

### CODE REVIEW

File: .github/workflows/hvm-ci.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        run: cd hvm && make coverage-update
+
+      - name: Commit updated coverage
+        uses: EndBug/add-and-commit@v9
+        with:
+          add: -A
+          message: commit from hvm-ci -- updated coverage
+
+      - name: Run tests
+        run: cd hvm && make coverage
```
Comment (hspitzley-czi) [inline .github/workflows/hvm-ci.yml:83]: Should this call `make test`?

**Review by hspitzley-czi — APPROVED** [review]

File: .github/workflows/hvm-ci.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    runs-on: [ARM64, self-hosted, Linux]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version-file: hvm/go.mod
+          cache: true
+      - name: Run go mod tidy
+        run: |
+          cd hvm && go mod tidy
```
[the remark this answers is not in the document]
Comment (jakeyheath) [reply .github/workflows/hvm-ci.yml:37]: something like: 

~~~
defaults:
      run:
        working-directory: ./app
~~~

File: hvm/COVERAGE [hunk 3]
```diff
@@ -0,0 +1 @@
+NaN
```
Comment (jakeyheath) [inline hvm/COVERAGE:1]: this doesn't seem right

**Review by jakeyheath — APPROVED** [review]

File: .github/workflows/hvm-ci.yml (hunk 1 above)
Comment (hspitzley-czi) [reply .github/workflows/hvm-ci.yml:83]: I guess just verify once you add tests that they're actually running 🤷

**Pull request merged**